framework_agreement/model/purchase.py

```diff
@@ -1,8 +1,5 @@
-# -*- coding: utf-8 -*-
```

Comment:
Don't remove this line please or someone with an accent in his name couldn't contribute to that file